tests/gc/gcleak3.nim

```diff
@@ -18,14 +18,10 @@ for i in 0..1024:
 
 proc limit*[t](a: var seq[t]) =
   var loop = s.len() - 512
-  for i in 0..high(int):
-    if i > high(s): break
-    #echo i
-    #GC_fullCollect()
+  while s.len > 0:
     if getOccupiedMem() > 3000_000: quit("still a leak!")
-    s.delete(i)
+    s.delete(high(s))
```

Comment:
but this (kind of) weakens the test; deleting the last element is "easier" (less work) than deleting elements 0 through high(int) and stopping when s is empty; that's why i preferred the `for i in 0..high(int):` way

Reply:
Excuse me, I wasn't sure about the intention of the test. It should be fixed now.